www/js/control/general-settings.js

```diff
@@ -131,6 +133,7 @@ angular.module('emission.main.control',['emission.services',
         $scope.settings.notification.prefReminderTime = m.format('LT'); // display in user's locale
         if (storeNewVal)
             NotificationScheduler.setReminderPrefs({ reminder_time_of_day: m.format('HH:mm') }); // store in HH:mm
+        $scope.settings.notification.scheduledNotifs = cordova.plugins.notification.local.scheduledNotifs;
```

Comment:
```suggestion
        $scope.settings.notification.scheduledNotifs = NotificationScheduler.scheduledNotifs;
```

@Abby-Wheelis Here's your fix! It needed to be changed here as well.